src/lib/y2users/config.rb

```diff
@@ -170,6 +184,9 @@ def detach_element(element)
       collection_for(element).delete(element.id) if exist
 
       element.assign_config(nil)
+
+      # Clean up the autologin user if the detached user was set for autologin
+      login.autologin_user = nil if login? && login.autologin_user&.is?(element)
```

Comment:
it is fine for me as long as it is documented in method about this side effect. Not sure if there is real use case for other behavior?

Reply:
No idea about other possible use cases. I think this would be the expected result, I will document it.